src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -138,6 +149,17 @@ private void throwDice(GameBean game) {
         game.setDiceThrown(true);
     }
 
+    private void buyCard(GameUserBean gameUser, GameBean game, Map<String, String> returnedParams) throws PlayException, GameException {
```

Comment:
1. when we changed some details, it was critical for particular story implementation. But now you are going to ADD (not "cahnge") new datails that have nothing to do with current story description. 
2. New details requires new planning, change of story scope, and creation of new tasks, such things should not be done during the sprint